lib/streamlit/__init__.py

```diff
@@ -479,7 +479,7 @@ def experimental_show(*args):
 
     except Exception:
         _, exc, exc_tb = _sys.exc_info()
-        exception(exc, exc_tb)  # noqa: F821
+        exception(exc)  # noqa: F821
```

Comment:
have you tried dropping `# noqa`s? I don't think we need them anymore.

Reply:
Done